### DISCUSSION TITLE
Impfnachweis für Genesene / Zertifikat für Erstimpfung

### DISCUSSION THREAD

**Discussion by msteffenDev** [Digitaler-Impfnachweis/certification-apis#136] [Q&A]
Wie soll die einmalige Impfung für einen Genesenen dokumentiert werden, der ja bekanntlicht nur eine Impfung bekommen wird? Welche Werte sind dann für Dosennummer und Gesamtdosen anzugeben? Wäre die Erkrankung gleichbedeutend mit der Erstimpfung?

Ist es zudem notwendig, dass bei Impfungen mit 2 Dosen sowohl ein Zertifikat für die Erstimpfung als auch eins für die Abschlussimpfung ausgestellt wird? Was wäre der Sinn dahinter? Eigentlich reicht doch das Zertifikat für die Abschlussimpfung aus.

**Comment by Ein-Tim** [comment]
Die gleiche Frage wird auch in #94 gestellt.

**Comment by ruiyan-yu** [comment]
Hallo,

@thinkberg 
Wir haben heute die Benachrichtigung über Vertragsänderung erhalten. Darin geht es um die Erstellung von Genesenen-Impfzertifikat.
Auf der linke Seite ist das bekannte Beispiel und auf der rechten Seite ist das wir gerade zum Testen erstellt haben.
Werden die PDF-Templates noch angepasst?
![DIFF](https://user-images.githubusercontent.com/18198895/124242616-3682ac00-db1d-11eb-9eff-ad7dbe6b5c16.png)


Mit freundlichen Grüßen

Duria e.G
